packages/purpose-process/test/updatePurpose.test.ts

```diff
@@ -137,7 +137,7 @@ describe("updatePurpose and updateReversePurpose", () => {
     await writeInReadmodel(toReadModelEService(eServiceDeliver), eservices);
     await writeInReadmodel(tenant, tenants);
 
-    await purposeService.updatePurpose({
+    const { purpose } = await purposeService.updatePurpose({
```

Comment:
this return value is not currently used. Was this intended to check the `isRiskAnalysisValid` value?